doc/source/user_guide/io.rst

```diff
@@ -1689,7 +1689,7 @@ The ``Series`` and ``DataFrame`` objects have an instance method ``to_csv`` whic
 allows storing the contents of the object as a comma-separated-values file. The
 function takes a number of arguments. Only the first is required.
 
-* ``path_or_buf``: A string path to the file to write or a StringIO
+* ``path_or_buf``: A string path to the file to write or a file object.  If a file object it must be opened with `newline=''`
```

Comment:
On all platforms, or only for Windows? (the issue seemed to be about windows, but didn't follow it)

Reply:
So in practice, it only matters on Windows, but I think it's safe advice to give on all platforms.  Python's csv docs do the same
https://docs.python.org/3/library/csv.html#csv.writer